### ISSUE TITLE
adjust rule paranoia level 

### ISSUE THREAD

**Issue by leveryd** [coreruleset/coreruleset#3272]
Hi all, i think below only based on keywords rule paranoia level should not be level 1.

https://github.com/coreruleset/coreruleset/blob/v3.3.4/util/regexp-assemble/regexp-942140.data

**Comment by azurit** [comment]
Ho @leveryd, have you experienced any false positives? Can you give an real-world example?

**Comment by leveryd** [comment]
@azurit yes, there is false positive in my situation, sorry for i can not give u complete request because of privacy,but i can tell u the matched argument part is below

```
jupiter的information_schema系统表权限
```

**Comment by leveryd** [comment]
Really? It is so obvious that it has no relationship with sql,I do not know why u think it is not a false postive.
And set only based-keyword rule paranoia level 1 is not reasonable, i think this is obvious too.

**Comment by azurit** [comment]
On the second thought, i agree this is a FP. We need to discuss what we can do with it.

**Comment by dune73** [comment]
We talked this through at our monthly meeting yesterday. We decided to leave it as is: https://github.com/coreruleset/coreruleset/issues/3252#issuecomment-1668414215

The reasoning is that the payload in question is an SQL related payload. It is a false positive but the use of the keyword was linked to SQL usage, so the rule hit is actually desired. It is a false positive, but in the same sense that a discussion about an SQL query would trigger false positives.

It is unfortunate and an edge case, but we want to keep this at PL1.

Closing this issue now.

**Comment by azurit** [comment]
@leveryd I can help you with writing an exclusion rule if you are interested.
